pom.xml

```diff
@@ -131,6 +131,12 @@
   </distributionManagement>
 
   <build>
+    <resources>
+      <resource>
+        <directory>src/main/resources</directory>
```

Comment:
Does this directory exist? I only see a `src/test/resources`. Is it just not committed to git?